local-cluster/tests/local_cluster.rs

```diff
@@ -256,19 +256,16 @@ fn run_network_partition(partitions: &[usize]) {
 }
 
 #[test]
-#[serial]
```

Comment:
This can have really weird behavior with local cluster tests. I wouldn't recommend unless you're confident it's stable like this. It's going to run multiple validators on the same system. It'll crush it.